compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -2132,24 +2303,42 @@ pub struct SourceScopeLocalData {
 ///////////////////////////////////////////////////////////////////////////
 // Operands
 
-/// These are values that can appear inside an rvalue. They are intentionally
-/// limited to prevent rvalues from being nested in one another.
+/// An operand in MIR represents a "value" in Rust, the definition of which is undecided and part of
+/// the memory model. One proposal for a definition of values can be found [on UCG][value-def].
+///
+/// [value-def]: https://github.com/rust-lang/unsafe-code-guidelines/blob/master/wip/value-domain.md
+///
+/// The most common way to create values is via a place to value conversion. A place to value
```

Comment:
Ah now this is the right place for the rant that I put earlier. ;)

I don't think we should all this a "conversion". That sounds like an `as` cast, like just putting something into a different representation. That's not at all what happens though. This is a *load* from memory, and should be described as that.